feature-pack/src/main/resources/content/bin/appclient.conf

```diff
@@ -42,7 +42,7 @@ fi
 # Specify options to pass to the Java VM.
 #
 if [ "x$JAVA_OPTS" = "x" ]; then
-   JAVA_OPTS="-Xms64m -Xmx512m -XX:MaxPermSize=256m -Djava.net.preferIPv4Stack=true"
```

Comment:
Only Sun's hotspot JVM used permgen and other JVMs tolerated XX:*PermSize parameters but didn't use them.
I removed it here as with JDK9 jvm fails to boot at all if it is present.